### ISSUE TITLE
[BUG] REVERSE_PROXY_URL results in too many redirects

### ISSUE THREAD

**Issue by sebstyle** [bunkerity/bunkerweb#839]
### What happened?

When configuring reverse proxy for the Bunkerweb UI omitting REVERSE_PROXY_URL will result in 404, setting REVERSE_PROXY_URL=/ will result in too many redirects.
The documentation does not suggest that REVERSE_PROXY_URL is mandatory.
My bunkerweb UI is only accessible from LAN and I prefer to use just the proxy.example.com without an additional /path

### How to reproduce?

Configuration taken from https://docs.bunkerweb.io/latest/web-ui/#advanced-installation
Replaced `proxy.example.com_REVERSE_PROXY_URL=/changeme` with `proxy.example.com_REVERSE_PROXY_URL=/` 

### Configuration file(s) (yaml or .env)

```YAML
version: "3.5"

services:
  bunkerweb:
    image: bunkerity/bunkerweb:1.5.4
    ports:
      - 80:8080
      - 443:8443
    labels:
      - "bunkerweb.INSTANCE=yes"
    environment:
      - SERVER_NAME=proxy.example.com
      - MULTISITE=yes
      - DATABASE_URI=mariadb+pymysql://bunkerweb:changeme@bw-db:3306/db # Remember to set a stronger password for the database
      - API_WHITELIST_IP=127.0.0.0/8 10.20.30.0/24
      - DISABLE_DEFAULT_SERVER=yes
      - USE_CLIENT_CACHE=yes
      - USE_GZIP=yes
      - proxy.example.com_USE_UI=yes
      - proxy.example.com_USE_REVERSE_PROXY=yes
      - proxy.example.com_REVERSE_PROXY_URL=/
      - proxy.example.com_REVERSE_PROXY_HOST=http://bw-ui:7000
      - proxy.example.com_INTERCEPTED_ERROR_CODES=400 404 405 413 429 500 501 502 503 504
    networks:
      - bw-universe
      - bw-services

  bw-scheduler:
    image: bunkerity/bunkerweb-scheduler:1.5.4
    depends_on:
      - bunkerweb
      - bw-docker
    environment:
      - DATABASE_URI=mariadb+pymysql://bunkerweb:changeme@bw-db:3306/db # Remember to set a stronger password for the database
      - DOCKER_HOST=tcp://bw-docker:2375
    networks:
      - bw-universe
      - bw-docker

  bw-docker:
    image: tecnativa/docker-socket-proxy:nightly
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
    environment:
      - CONTAINERS=1
      - LOG_LEVEL=warning
    networks:
      - bw-docker

  bw-ui:
    image: bunkerity/bunkerweb-ui:1.5.4
    depends_on:
      - bw-docker
    environment:
      - DATABASE_URI=mariadb+pymysql://bunkerweb:changeme@bw-db:3306/db # Remember to set a stronger password for the database
      - DOCKER_HOST=tcp://bw-docker:2375
      - ADMIN_USERNAME=changeme
      - ADMIN_PASSWORD=changeme # Remember to set a stronger password for the changeme user
    networks:
      - bw-universe
      - bw-docker

  bw-db:
    image: mariadb:10.10
    environment:
      - MYSQL_RANDOM_ROOT_PASSWORD=yes
      - MYSQL_DATABASE=db
      - MYSQL_USER=bunkerweb
      - MYSQL_PASSWORD=changeme # Remember to set a stronger password for the database
    volumes:
      - bw-data:/var/lib/mysql
    networks:
      - bw-docker

volumes:
  bw-data:

networks:
  bw-universe:
    name: bw-universe
    ipam:
      driver: default
      config:
        - subnet: 10.20.30.0/24
  bw-services:
    name: bw-services
  bw-docker:
    name: bw-docker
```

### BunkerWeb version

1.5.4

### What integration are you using?

Docker

### Linux distribution (if applicable)

Debian 12

### Removed private data

- [X] I have removed all private data from the configuration file and the logs

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fl0ppy-d1sk** [comment]
Hello @sebstyle,

Thanks for your report, this will be fixed in 1.5.5

**Comment by fl0ppy-d1sk** [comment]
Fixed in the v1.5.5 release !

Thanks @sebstyle
